config.yaml

```diff
@@ -1,28 +1,63 @@
 destination: &dest syn12177492
 staging_path: ./staging
+sources:
+  - genes_biodomains:
+    genes_biodomains_files: &genes_biodomains_files
+      - name: genes_biodomains
+        id: syn44151254.1
+        format: csv
+    genes_biodomains_provenance: &genes_biodomains_provenance
+      - syn44151254.1
+  - overall_scores:
+    overall_scores_files: &overall_scores_files
+      - name: overall_scores
+        id: syn25575156.13
+        format: table
+    overall_scores_provenance: &overall_scores_provenance
+      - syn25575156.13
+  - agora_proteomics:
+    agora_proteomics_files: &agora_proteomics_files
+      - name: proteomics
+        id: syn18689335.3
+        format: csv
+    agora_proteomics_provenance: &agora_proteomics_provenance
+      - syn18689335.3
+  - agora_proteomics_tmt:
+    agora_proteomics_tmt_files: &agora_proteomics_tmt_files
+      - name: proteomics_tmt
+        id: syn35221005.2
+        format: csv
+    agora_proteomics_tmt_provenance: &agora_proteomics_tmt_provenance
+      - syn35221005.2
+  - agora_proteomics_srm:
+    agora_proteomics_srm_files: &agora_proteomics_srm_files
+      - name: proteomics_srm
+        id: syn52579640.3
+        format: csv
+    agora_proteomics_srm_provenance: &agora_proteomics_srm_provenance
+      - syn52579640.3
+  - rna_distribution_data:
```

Comment:
This should be called `rna_diff_exp_data` (+ `rna_diff_exp_data_files`) instead, to avoid confusion. We have some oddness with rna distribution vs. diff exp data. Diff exp data is the input, and one of the transforms outputs distribution data (`rna_distribution_data`) and the other two use the plain diff exp data as-is (`gene_info` and `rnaseq_differential_expression`).

Reply:
updated in latest commit